src/pages/Productdetail/ImageSlider.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import './ImageSlider.scss';
+
+class ImageSlider extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      sliderIndex: 0,
+      productSlideImg: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/productDetail.json', {})
```

Comment:
```suggestion
    fetch('/data/productDetail.json')
```
1. 옵션에 아무것도 안 넣을거라면 굳이 인자로 전달 안하셔도 됩니다
2. localhost:3000은 안적어주셔도 됩니다 그냥 바로 절대경로로 시작하게 된다면 CRA에서 자동으로 현재 프로젝트가 돌아가고 있는 서버주소로 인식해줍니다!, 오히려 주소를 명시적으로 적어준다면 localhost:3000이 아닌 다른 주소에서 실행 됬을 경우 문제가 생길 수 있겠네요

Reply:
mockdata 하나로 묶어서 해당 부분 삭제했습니다:) 새로 만든 mockdata에 로컬호스트 부분 삭제했습니다